runtime/volta/src/lib.rs

```diff
@@ -663,6 +664,9 @@ impl pallet_token_claim::Config for Runtime {
     type MaxBeneficiaries = MaxBeneficiaries;
     type MaxClaimMessageLength = MaxClaimMessageLength;
     type AccountIdBytesToSign = pallet_token_claim::AccountId32ToSs58BytesToSign;
+    type MaxOpBeneficiaries = MaxOpBeneficiaries;
```

Comment:
Minor thing: we might also add a static assertion to check the equality of `MaxOpBeneficiaries` and `MAX_OP_BENEFICIARIES`, similarly to what we do at lines 676-680